api.swagger.yaml

```diff
@@ -3386,6 +3432,18 @@ paths:
           schema:
             type: string
             example: '2012-12-25'
+        - description: 'Filter by external ID.'
+          in: query
+          name: external_id
+          schema:
+            type: string
+            example: 'ABC123'
+        - name: state
+          description: 'Filter the list of lists on a comma-separated list of states. Valid statuses are "draft", "needs_review", "reviewed", "published", and "unpublished". You may include multiple. The default when not supplied is "published" only.'
```

Comment:
Not sure this should be in this PR? If it should then 'lists' should be 'ideas'

Reply:
chopped out